examples/CharsetInput/CharsetInput.ino

```diff
@@ -20,7 +21,8 @@ char charset[] = {'0', '1', '2', '3', '4', '5', '6', '7', '8', '9'};
 void inputCallback(char *value);
 
 MAIN_MENU(
-    ITEM_INPUT("Con", inputCallback),
+    new ItemInput("Con", "", inputCallback),
+    new ItemInputCharset("Con2", "", charset, 10, inputCallback),
```

Comment:
Please use the macro for `ITEM_INPUT`
and add a macro for ItemInputCharset, doesn't look good mixing stuff, the macros were created so that you don't have to type `new` everytime.
```suggestion
    ITEM_INPUT("Con", "", inputCallback),
    ITEM_INPUT_CHARSET("Con2", "", charset, 10, inputCallback),
```